.vscode/launch.json

```diff
@@ -33,7 +33,8 @@
                 "${file}"
             ],
             "env": {
-                "BICEP_TRACING_ENABLED": "true"
+                "BICEP_TRACING_ENABLED": "true",
+                "BICEP_REGISTRY_FQDN": "asilvermantestbr.azurecr.io"

```

Comment:
Since this is a personal resource, I don't think this should be committed to `main`.